src/functionalTest/groovy/com/autonomousapps/AbstractFunctionalSpec.groovy

```diff
@@ -59,15 +59,17 @@ abstract class AbstractFunctionalSpec extends Specification {
   }
 
   protected static boolean isCompatible(GradleVersion gradleVersion, AgpVersion agpVersion) {
+    // TODO can't find docs on min Gradle version for AGP 7
+
     if (agpVersion >= AgpVersion.version('7.0.0')) {
-      // TODO can't find docs on min Gradle version.
       return gradleVersion >= GradleVersion.version('6.8.0')
```

Comment:
nit: AGP 7.x is meant to work with Gradle 7.x APIs

https://android-developers.googleblog.com/2020/12/announcing-android-gradle-plugin.html

Reply:
Sure, but in my initial testing it's not an auto fail with 6.8, which is interesting.